app/webpacker/lib/wca-data.js.erb

```diff
@@ -245,3 +245,6 @@ export const cronjobs = <%= JobUtils::WCA_CRONJOBS.map(&:to_s).to_h { |item| [it
 
 // ----- RESULTS -----
 export const duplicateCheckerJobRunStatuses = <%= DuplicateCheckerJobRun.run_statuses.to_json %>
+
+// ----- PRODUCTION CONFIG -----
+export const isProduction = <%= Rails.env.production? && EnvConfig.WCA_LIVE_SITE? %>
```

Comment:
I'm afraid I have to veto exporting `Rails.env` specific stuff, because there won't be any equivalent of exporting this value to NextJS.

Can you think of another way, for example by passing some boolean flag (that determines whether the button should be shown or not) to the props of the component via `react_component` view helper?

Reply:
Okay how about making it viewable only in adminView?

Reply:
For now, that makes sense. In the future, we obviously also want to launch it for the public.